java/test/jmri/implementation/DefaultConditionalTest.java

```diff
@@ -275,6 +275,24 @@ public void testCalculate() {
         ix1.setStateVariables(conditionalVariablesList_FalseFalseFalse);
         Assert.assertTrue("calculate() returns NamedBean.FALSE", ix1.calculate(false, null) == Conditional.FALSE);
     }
+    
+    @Test
+    public void testTriggers() {
```

Comment:
Just a hint, if you have a test that isn’t ready yet, you can mark it with an ignore annotation.  JUnit4 will skip the test during execution.